pop_tools/eos.py

```diff
@@ -94,10 +94,8 @@ def eos(salt, temp, return_coefs=False, **kwargs):
     smax = 999.0
 
     if use_xarray:
-        temp = xr.where(temp < tmin, tmin, temp)
-        temp = xr.where(temp > tmax, tmax, temp)
-        salt = xr.where(salt < smin, smin, salt)
-        salt = xr.where(salt > smax, smax, salt)
+        temp = temp.clip(tmin, tmax)
+        salt = salt.clip(smin, smax)
```

Comment:
It's my understanding that 

Line 97-98

```python
temp = temp.clip(tmin, tmax)
salt = salt.clip(smin, smax)
```

and 

Line 134-135

```python
temp = np.clip(temp, tmin, tmax)
salt = np.clip(salt, smin, smax)
```

are doing exactly the same thing, and can be consolidated into just 

```python
temp = np.clip(temp, tmin, tmax)
salt = np.clip(salt, smin, smax)
```

which will return an xarray object or a numpy array depending on the type of the input.

Reply:
Never mind ;) https://github.com/NCAR/pop-tools/pull/63#issue-478211186

Reply:
There is one unfortunate difference: https://github.com/pydata/xarray/issues/2570

Can we put a min numpy of 1.17? I'm not sure it affects anyone really and saves some effort.  1.17.0 was released in June 2019.